content/_data/changelogs/weekly.yml

```diff
@@ -392,6 +392,35 @@
         SSH CLI client authenticator 1.3. 
         Expose <code>PublicKeySignatureWriter</code> to plugins.
       pull: 2840
+- version: "2.56"
+  date: 2017-04-23
+  changes:
+    - type: rfe
+      message: >
+        Plugins may not have been expecting <code>InvalidPathException</code> since 2.55, so wrap then in <code>IOException</code> to restore previous behavior.
```

Comment:
I've forgotten to edit that. IMHO the core broke plugins by undocumented runtime exceptions.

I would just write "Prevent InvalidPathExceptions when passing invalid paths to the File handling methods within the core (regression in 2.55)"